### PR TITLE
feat: snowflake CRUD + emailing category (read, clean, respond)

### PR DESCRIPTION
**Pull request by sanjaysabu4205**

### CODE REVIEW

**Comment by sanjaysabu4205** [comment]
Snowflake using CRUD method

File: Database/Snowflake/Execute query.ipynb [hunk 1]
```diff
@@ -1,206 +0,0 @@
-{
```
Comment (BobCashStory) [inline Database/Snowflake/Execute query.ipynb:1]: please don't use space in file naming, use `_` instead

**Review by BobCashStory — COMMENTED** [review]
Hey @sanjaysabu4205 thanks for your Pr if you can just update the name as i commented we will be happy to merge your work

File: Database/Snowflake/Execute query.ipynb (hunk 1 above)
Comment (sanjaysabu4205) [reply Database/Snowflake/Execute query.ipynb:1]: ok

**Review by riderx — APPROVED** [review]

**Pull request merged**